requirements.txt

```diff
@@ -1,5 +1,5 @@
 scipion-pyworkflow>=3.0.15
-biopython==1.79
-tifffile<=2021.11.2
+biopython==1.80
+tifffile<=2023.1.23.1
 emtable
-scipy<=1.8.1 # Fix for dependency conflict with PyWavelets --> scipy 1.9 --> numpy > pyworkflow numpy
+scipy<=1.10.0
```

Comment:
Why removing this: scipy<=1.8.1 # Fix for dependency conflict with PyWavelets --> scipy 1.9 --> numpy > pyworkflow numpy

Reply:
@pconesa xmipp supports up to 1.10.0: https://github.com/I2PC/scipion-em-xmipp/blob/devel/requirements.txt#L4C1-L4C14